prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+"""
+CP1404/CP5632 Practical
+Finding color code
+Wimbledon
+Estimate: 30 minutes
+Actual:   35 minutes
+"""
+CHAMPION_INDEX = 2
+COUNTRY_INDEX = 3
```

Comment:
I believe the winning country index is actually 1. 3 is the index for the runner up countries.